actions/action-lint/README.md

```diff
@@ -1,20 +1,4 @@
 # action-lint
 
-This action will lint all actions and workflows of a repository.
-
-## Input Parameters
-
-| Name                | Required | Default Value |  Type  | Description                                |
-| ------------------- | :------: | :-----------: | :----: | ------------------------------------------ |
-| ref                 |    ❌    |      ""       | string | The ref name to checkout the repository.   |
-| action-lint-version |    ❌    |    v1.6.22    | string | The action lint repository version to use. |
-
-## Usage
-
-```yaml
-steps:
-  - name: Lint actions and workflows
-    uses: bakdata/ci-templates/actions/action-lint@main
-    with:
-      ref: "my-awesome-ref" # (Optional)
-```
+- [Description](https://github.com/bakdata/ci-templates/tree/feat/doc/docs/descriptions/actions/action-lint)
+- [References](https://github.com/bakdata/ci-templates/tree/feat/doc/docs/references/actions/action-lint)
```

Comment:
If you want to you can also just put links in the Readme.md in the root folder to tell people that docs for actions and workflows are in the docs folder instead of having links in every action path